### ISSUE TITLE
Improve keyboard accessiblity on statusBar

### ISSUE THREAD

**Issue by j264415** [jupyterlab/jupyterlab#14852]
### Problem

1. When testing keyboard navigation the **line column**, **kernel status and running session** components were not reachable when tabbing. 

2. Furthermore, these components do not respond to **"Enter"** and **"Spacebar"** . 

### Proposed Solution

To increase keyboard navigation accessibility for users who need it, we suggest adding **"tabIndex=0"** to: **the line/Column, the Kernel status and, the running session** components in the statusBar. 

Additionally, keyboard functionality should be added so that these elements will respond to **"Enter"** and **"Spacebar"**. 

### Additional context

Working on a ticket for this issue.

**Comment by krassowski** [comment]
Closing as resolved by https://github.com/jupyterlab/jupyterlab/pull/14853
